src/Parser/Rogue/Subtlety/Modules/RogueCore/ComboPointTracker.js

```diff
@@ -1,15 +1,31 @@
+import SPELLS from 'common/SPELLS';
 import Combatants from 'Parser/Core/Modules/Combatants';
+import SpellUsable from 'Parser/Core/Modules/SpellUsable';
 
 import ResourceTracker from '../ResourceTracker/ResourceTracker';
 
 class ComboPointTracker extends ResourceTracker {
   static dependencies = {
     combatants: Combatants,
+		spellUsable: SpellUsable,
   };
 
+  danceReduction;
+
   on_initialized() {
     this.resourceType = 4;
     this.resourceName = "Combo Points";
```

Comment:
While it probably doesn't matter, resourceType and resourceName probably should be set baseline, rather than in on_initialized()